vms/registration/tests/test_functional_admin.py

```diff
@@ -240,9 +268,9 @@ def test_address_field(self):
         self.driver.find_element_by_id('id_last_name').send_keys('last-name')
         self.driver.find_element_by_id('id_email').send_keys('email2@systers.org')
         self.driver.find_element_by_id('id_address').send_keys('admin-address!@#$()')
-        self.driver.find_element_by_id('id_city').send_keys('admin-city')
-        self.driver.find_element_by_id('id_state').send_keys('admin-state')
-        self.driver.find_element_by_id('id_country').send_keys('admin-country')
+        self.driver.find_element_by_id('id_city').send_keys('!$@%^#&admin-city')
+        self.driver.find_element_by_id('id_state').send_keys('!$@%^#&admin-state')
```

Comment:
@tapasweni-pathak This is for checking regex validation/ legit characters for a field as per the model. In this part of the test, I am checking what would happen if I enter special characters  in city, state and country. I should get a warning for each of these fields then that these are not valid characters and registration shouldn't be allowed